.buildkite/scripts/install_tools.sh

```diff
@@ -32,3 +37,14 @@ fi
 with_go "${GO_VERSION}"
 with_mage
 with_python
+
+#sudo command doesn't work at the "pre-command" hook
+#sudo chmod -R go-w "${BEATS_PROJECT_NAME}/"     #fix the fulesystem permissions issue like this:https://buildkite.com/elastic/beats-metricbeat/builds/1154#018d12db-dc0c-4bcd-b9b4-d5dece0b42c6/272-1267
```

Comment:
<img width="1091" alt="image" src="https://github.com/elastic/beats/assets/2871786/e405cbc8-0ed3-4d6b-ba9e-16625e652a65">

```
=== FAIL: metricbeat/mb TestProcessorsForMetricSet_ProcessorsRead (0.00s)
--
  | lightmodules_test.go:424:
  | Error Trace:	/opt/buildkite-agent/builds/bk-agent-prod-gcp-1705418421998491635/elastic/beats-metricbeat/metricbeat/mb/lightmodules_test.go:424
  | Error:      	Received unexpected error:
  | reading processors for metricset 'withprocessors' in module 'unpack': loading light module 'unpack' definition: loading module configuration from 'testdata/lightmodules/unpack/module.yml': config file ("testdata/lightmodules/unpack/module.yml") can only be writable by the owner but the permissions are "-rw-rw-r--" (to fix the permissions use: 'chmod go-w /opt/buildkite-agent/builds/bk-agent-prod-gcp-1705418421998491635/elastic/beats-metricbeat/metricbeat/mb/testdata/lightmodules/unpack/module.yml')
  | Test:       	TestProcessorsForMetricSet_ProcessorsRead
```

I'm certainly confused what's the reason for this. IIUC, the Buildkite workspace owner, and the user running the shell/command steps in the Buildkite pipeline should match. Should this issue be reported to the CI Systems owners if that's not the case?

Or does it happen if you run the same commands locally in a vanilla GitHub clone? Just wanna be sure that the build system is acting wrongly for some reason.